src/moveFile.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function move(dir, file, data) {
+  fs.mkdir(dir, { recursive: true }, (e) => {
```

Comment:
The app must throw an error if the destination directory does not exist, but it is creating the directory if it doesn't exist. Remove the 'recursive: true' option and handle the error if the directory does not exist.